### ISSUE TITLE
Experimental PPR cannot be used in stable Next 15 (but mentioned in docs)

### ISSUE THREAD

**Issue by neoromantic** [vercel/next.js#71587]
### Link to the code that reproduces this issue

https://github.com/neoromantic/next-15-ppr-reproduction

### To Reproduce

1. Install last Next.js with create-next-app
2. Add  configuration to next.config.ts:

```
  experimental: {
    ppr: 'incremental',
  },
```
3. Run build or dev server

### Current vs. Expected behavior

If we run Next 15.0.0 stable with ppr enabled, we get [Canary Error](https://github.com/vercel/next.js/blob/475bdb1f743bf0b3552b40f85b615d87f4ae4904/packages/next/src/server/config.ts#L247)

However, Docs suggests that it is supported: https://nextjs.org/docs/app/building-your-application/rendering/partial-prerendering with no mention of Canary versions.

It either should allow PPR in stable, or docs should be adjusted to not suggest using PPR in Next 15 (since it was released)

### Provide environment information

```bash
N/A
```

### Which area(s) are affected? (Select all that apply)

Documentation, Partial Prerendering (PPR)

### Which stage(s) are affected? (Select all that apply)

next dev (local), next build (local), next start (local)

**Comment by alecaserez** [comment]
I have the same problem. After migrating my entire project I wanted to try this new feature and it was not possible.

**Comment by mephmanx** [comment]
Also having this issue...

**Comment by neoromantic** [comment]
> I have the same problem. After migrating my entire project I wanted to try this new feature and it was not possible.

You just have to move to canary version. My issue is that docs are implying that Next 15 stable supports it, which it does not.

**Comment by alecaserez** [comment]
> > I have the same problem. After migrating my entire project I wanted to try this new feature and it was not possible.
> 
> You just have to move to canary version. My issue is that docs are implying that Next 15 stable supports it, which it does not.

Looking at the documentation, I thought the intention was that it is already supported in the standard version. Maybe that's the intention.

**Comment by samcx** [comment]
@neoromantic @alecaserez Yes, it is only supported in canary still at the moment. We are working to fix the stable docs!

**Comment by finfin** [comment]
https://nextjs.org/learn/dashboard-app/partial-prerendering the learning material should be updated as well. 

The guide uses stable next.js, hence will encounter the same issue. And it's not an easy task to switch to canary version when we go through to course midway.

**Comment by huozhi** [comment]
We tweaked the docs and add the disclaimer about canary retriction, the oneline docs will be updated very soon! Thanks
